scripts/config.ts

```diff
@@ -0,0 +1,16 @@
+import { ConfigData } from "./@types/ConfigData";
+
+export const config: ConfigData = {
+    // the data of the Figma file we sync with
+    figmaFile: {
+        // this is the "production" file
+        id: 'TLnoT5AYQfy3tZ0H68BgOr',
```

Comment:
non-blocking question: do we want this to be a .env secret instead?

Reply:
why? it's not a secret thing. is an ID of a Figma file (that has its own permissions, so even if a user knows the ID, they can't do anything with it). 

@Dhaulagiri any thoughts on this?

Reply:
we had a bit of back and forth on this before you joined and while we did move it to to an `.env` we ultimately agreed with what you are saying here. If it doesn't seem like something needs to be configurable I think this is ok.